Package.swift

```diff
@@ -1028,6 +1034,8 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(url: "https://github.com/apple/swift-collections.git", "1.0.1" ..< "1.2.0"),
         .package(url: "https://github.com/apple/swift-certificates.git", "1.0.1" ..< "1.6.0"),
         .package(url: "https://github.com/swiftlang/swift-toolchain-sqlite.git", from: "1.0.0"),
+        // For use in previewing documentation
+        .package(url: "https://github.com/swiftlang/swift-docc-plugin", from: "1.1.0"),
```

Comment:
What's the reason for unconditionally including this dependency? If I'm building SwiftPM only for development or testing, this will still be cloned, with its targets included in build manifests, making the build slower. What happened to the environment variable that only conditionally included this dependency?